flang/lib/Optimizer/Dialect/FIRType.cpp

```diff
@@ -1557,3 +1377,95 @@ bool fir::isa_unknown_size_box(mlir::Type t) {
   }
   return false;
 }
+
+namespace fir {
```

Comment:
Nit: I believe the LLVM coding style says not to include function definitions in an extended namespace like this. Instead, each function's name is supposed to be prefixed with the entire namespace.

Reply:
Ok I'll update it then.